### ISSUE TITLE
Add information about Asset 223

### ISSUE THREAD

**Issue by adamsteeber** [opensquare-network/statescan#1035]
Image:

![bill](https://user-images.githubusercontent.com/79496485/158116063-f985f1b0-a744-4daa-9c50-28b052cd6130.png)

About can be "For educational purposes only"

**Comment by wliyongfeng** [comment]
Hey @adamsteeber, thanks for this issue, but can you represent Bill to post his photo? Or do you have the approval from Bill?

**Comment by adamsteeber** [comment]
I don't represent him but I thought I would take the liberty to suggest it for the aesthetic :) But you're right, Bill should approve first. If we can get Bill in here to get his thoughts that would be ideal!

**Comment by wliyongfeng** [comment]
Of course. I appreciated the suggestion, but we'd better have Bill's approval or Bill has announced this logo.

**Comment by Popoulosss** [comment]
> I don't represent him but I thought I would take the liberty to suggest it for the aesthetic :) But you're right, Bill should approve first. If we can get Bill in here to get his thoughts that would be ideal!

Hi @adamsteeber, If you get Bill's approval, you can submit the information need to update [here](https://docs.google.com/forms/d/e/1FAIpQLSfoMHYxCq17xSAyEiYiXNqcfGF-PPWYTJf1yIkzw1iNR22jyA/viewform)(Google Form), refer to [RMRK](https://statemine.statescan.io/asset/8).

**Comment by laboon** [comment]
Oi, is that a bad picture of me!  Why don't you use the Billcoin logo that sub.id ( https://sub.id/ ) uses? 
![Billcoin_Icon_Dark](https://user-images.githubusercontent.com/1417717/160369879-b5ebaa56-5e51-48bd-be7a-70f0d6260689.svg)
![Billcoin_Icon_Light](https://user-images.githubusercontent.com/1417717/160369885-0eebd96e-59a0-4f2d-8501-25b89ba5896b.svg)

**Comment by laboon** [comment]
I technically say Billcoin is "just for fun" but "educational purposes only" is also fine.

**Comment by wliyongfeng** [comment]
Thanks @laboon , will update it with the one on sub.id soon.

**Comment by Popoulosss** [comment]
[Replace asset icon](https://statemine.statescan.io/asset/223)

files
[icons-asset-bill.zip](https://github.com/opensquare-network/statescan/files/8362107/icons-asset-bill.zip)
